src/sensor-fusion/fusion-pose-sensor.js

```diff
@@ -17,6 +17,11 @@ import PosePredictor from './pose-predictor.js';
 import * as MathUtil from '../math-util.js';
 import * as Util from '../util.js';
 
+// Frequency which the Sensors will attempt to fire their
+// `reading` functions at. Use 16.6ms to achieve 60FPS, since
+// we generally can't get higher without native VR hardware
+const SENSOR_FREQUENCY = 16.6;
```

Comment:
this frequency, in my testing, seems to be hit a real nice sweet spot on both my Samsung Galaxy S8 and S6. very impressive.